src/dodal/devices/i19/diffractometer.py

```diff
@@ -0,0 +1,28 @@
+from ophyd_async.epics.motor import Motor
+
+from dodal.devices.motors import Stage, XYZStage
+
+CIRC = "-MO-CIRC-02:SAM:"  # for phi, kappa, omega, 2theta and det_z
+SAMP = "-MO-SAMP-02:SAM:"  # for x,y,z
```

Comment:
Should: @DiamondJoseph has gone through and pulled these out into parameters to pass to the device (https://github.com/DiamondLightSource/dodal/pull/1388/files). I think that is a good standard and we should do the same here